packages/L10n/src/locales/es.yml

```diff
@@ -189,7 +189,37 @@ es:
     copyInput:
       clicked_tooltip: Copiado
       hover_tooltip: Copiar al portapapeles
-    dynamicHyperlink:
+    dynamicHyperlinkTransformer:
       access_denied: "Acceso denegado"
       invalid_url: "URL no válida"
       loading: Cargando...
+    filter:
+      actions:
+        add: Agregar filtro
+        clear: Borrar
+        delete: Eliminar
+      no_filters_applied: No se aplicaron filtros a esta vista
+      label: Filtrar
+      lowercase_label: "{{count}} filtro"
+      lowercase_label_plural: "{{count}} filtros"
+      and: Y
+      or: O
+      rules:
+        is: es
+        is_not: no es
+        contains: contiene
+        does_not_contain: no contiene
+        is_blank: está en blanco
+        is_not_blank: no está en blanco
+        equals: es igual a
+        not_equal_to: es distinto de
```

Comment:
`no igual a` - seems better for consistency. thoughts?

Reply:
We could use "no es igual a" for consistency, but I still prefer "es distinto de".
What do you think, @cristinajimenez?

Reply:
Agree, if we can choose, I'd rather use `es distinto de`

Reply:
...but note that it will be quite different than what we have in docs, Academy, AN, etc. That's why I meant to use `no es igual a`. ..and we don't want to edit legacy content in that case.

Reply:
![image](https://user-images.githubusercontent.com/4226672/105519457-49e83c00-5c8e-11eb-9a4a-3f9a9d1514be.png)

Reply:
![image](https://user-images.githubusercontent.com/4226672/105519563-697f6480-5c8e-11eb-8015-e6ad40b8e25b.png)

Reply:
Sure. We can use `no es igual a` for consistency with other TMs...